console/services/config_service.py

```diff
@@ -382,5 +387,20 @@ def add_config_without_reload(self, key, default_value, type, desc=""):
         else:
             raise ConfigExistError("配置{}已存在".format(key))
 
+    def get_upgrade_status(self, key):
```

Comment:
Whether you need to upgrade depends on the data version or the code version. It should not be a configuration value.